metric_learn/lsml.py

```diff
@@ -134,16 +134,22 @@ def _gradient(self, metric):
 class LSML_Supervised(LSML):
   def __init__(self, tol=1e-3, max_iter=1000, prior=None, num_labeled=np.inf,
                num_constraints=None, weights=None, verbose=False):
-    """Initialize the learner.
+    """Initialize the supervised version of `LSML`.
+
+    `LSML_Supervised` creates quadruplets from labeled samples by taking two
+    samples from the same class, and two samples from different classes, and
+    concatenating them. This way it builds quadruplets where the two first
```

Comment:
", and concatenating them" seems useless

Reply:
Agreed
Done